schema/vrs.yaml

```diff
@@ -191,7 +191,7 @@ definitions:
           - $ref: "#/definitions/SequenceExpression"
           - $ref: "#/definitions/CURIE"
       copies:
-        $ref: "#/definitions/Range"
+        $ref: "#/definitions/IntegerOrRange"
```

Comment:
My preference would be to just inline the oneOf. Why? Because it (IMO) reduces a level of indirection when reading the using class.

That is, the docs for this class will read one of two ways.
1) `copies: IntegerOrRange` (and you have to go see what that means)
2) `copies: Integer | IntegerRange`

The latter is more transparent.

Reply:
Agreed.